docs/doxygen/ie_docs.xml

```diff
@@ -481,23 +481,22 @@
                     <tab type="user" title="Readers" url="@ref omz_tools_accuracy_checker_accuracy_checker_data_readers_README"/>
                     <tab type="user" title="Caffe* Installation Tips" url="@ref omz_tools_accuracy_checker_accuracy_checker_launcher_caffe_installation_readme"/>
                 </tab>
-                <tab type="usergroup" title="Post-Training Optimization Tool" url="@ref pot_README">
+                <tab type="usergroup" title="Post-Training Optimization Toolkit" url="@ref pot_README">
                   <tab type="usergroup" title="Quantization" url="@ref pot_compression_algorithms_quantization_README">
                     <tab type="user" title="DefaultQuantization Algorithm" url="@ref pot_compression_algorithms_quantization_default_README"/>
                     <tab type="user" title="AccuracyAwareQuantization Algorithm" url="@ref pot_compression_algorithms_quantization_accuracy_aware_README"/>
                     <tab type="user" title="TunableQuantization Algorithm" url="@ref pot_compression_algorithms_quantization_tunable_quantization_README"/>
                   </tab>
-                  <tab type="usergroup" title="Quantization Hyperparameters Optimization" url="@ref pot_compression_optimization_README">
-                    <tab type="usergroup" title="Tree-structured Parzen Estimator (TPE)" url="@ref pot_compression_optimization_tpe_README">
+                  <tab type="usergroup" title="Global Optimization" url="@ref pot_compression_optimization_README">
+                    <tab type="usergroup" title="Tree-Structured Parzen Estimator (TPE)" url="@ref pot_compression_optimization_tpe_README">
                       <tab type="user" title="TPE Multiple Node Configuration Based on MongoDB Database" url="@ref pot_compression_optimization_tpe_multinode"/>
                     </tab>
                   </tab>
-                  <tab type="usergroup" title="Low Precision Optimization Guide" url="@ref pot_docs_LowPrecisionOptimizationGuide">
-                    <tab type="user" title="Post-Training Optimization Best Practices" url="@ref pot_docs_BestPractices"/>
-                  </tab>
+                  <tab type="user" title="Low Precision Optimization Guide" url="@ref pot_docs_LowPrecisionOptimizationGuide"/>
+                  <tab type="user" title="Post-Training Optimization Best Practices" url="@ref pot_docs_BestPractices"/>
                   <tab type="user" title="Use Post-Training Optimization Toolkit API" url="@ref pot_compression_api_README"/>
-                  <tab type="usergroup" title="Define Configuration Files" url="@ref pot_configs_README">
-                    <tab type="user" title="Examples of Configuration Files" url="@ref pot_configs_examples_README"/>
+                  <tab type="usergroup" title="Configuration File Description" url="@ref pot_configs_README">
+                    <tab type="user" title="Run Examples" url="@ref pot_configs_examples_README"/>
```

Comment:
I guess this one should be rather "How To Run Examples" or something similar, the same change is needed in POT docs.

Reply:
Ok, corrected. Actually I think we should discuss the structure and the titles in the team so this is why I haven't really made any changes, just aligned. In my view it should be improved.

Reply:
From the user's point of view, it might not look clear, which examples are mentioned. As the topic describes running POT for a particular sample model, I'd suggest changing the title to one of the following:
- Usage Examples
- Run Post-Training Optimization Tool with Sample Configurations

Reply:
Andrey, it's under "Examples of Configuration Files" so it should be clear which examples are mentioned. 
As I said above, we will need to discuss the structure and the titles in the team and at this moment I just want to align the menu and the titles to address one of the DX/UX comments.